### PR TITLE
export all supported formats

### PR DESCRIPTION
**Pull request by tw4452852**
Signed-off-by: Tw <tw19881113@gmail.com>

### CODE REVIEW

File: rar.go [hunk 1]
```diff
@@ -10,8 +10,12 @@ import (
 	"github.com/nwaples/rardecode"
 )
 
+// Rar is for RAR archive format
+var Rar Archiver
```
Comment (mholt) [inline rar.go]: Can you just initialize this here, instead of in init? `var Are = RarFormat` for this one, and similarly for the other formats.

Comment (tw4452852) [inline rar.go]: Done.

**Comment by tw4452852** [comment]
Hi @mholt , could we just remove go1.6 in CI, it's just noisy...

**Comment by mholt** [comment]
Yes, that's fine. I'm mobile or I would just do it right now. Good idea.

**Pull request merged**